test/common/benchmarks.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
```

Comment:
I believe this will need a `/* eslint-disable required-modules */` at the top to pass linting. 
Please run `make lint` to make sure :-)